src/main/java/core/basesyntax/AbstractFigure.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public abstract class AbstractFigure implements Drawable, AreaCalculator, Colorable {
+    protected Color color;
+
+    public AbstractFigure(Color color) {
+    }
```

Comment:
For what you need this constructor

Reply:
you ignore color that is passed, you should assign field to it